gamefixes-umu/umu-321040.py

```diff
@@ -0,0 +1,9 @@
+"""Game fix for Dirt 3 Complete Edition"""
+
+
+from protonfixes import util
+
+
+def main() -> None:
+    """Installs OpenAL library, without it the game simply wont launch on proton 8 or above"""
+    util.protontricks('Openal')
```

Comment:
Instead of a file, this should be a symbolic link to the corresponding Steam fix

Reply:
This needs to be deleted. Remove the file locally then commit and force push your changes.